mockApis/auth.ts

```diff
@@ -214,11 +215,11 @@ export default class AuthServiceMocks {
   }
 
   stubServiceProviderToken = async (): Promise<unknown> => {
-    return this.stubToken('delius')
```

Comment:
you have no idea how long it took me to find this bug. 

i don't know how these tests were passing before - but i find it reassuring that this change set surfaced this.